tests/test_enterprise/api/test_filters.py

```diff
@@ -223,7 +223,19 @@ def test_filter(self, is_staff, is_linked_to_enterprise, has_access):
             for key, value in self.enterprise_customer_data.items():
                 assert enterprise_customer_response[key] == value
         else:
-            assert response == {'count': 0, 'next': None, 'previous': None, 'results': []}
+            mock_empty_200_success_response = {
+                'next': None,
+                'previous': None,
+                'count': 0,
+                'num_pages': 1,
+                'current_page': 1,
+                'start': 0,
```

Comment:
I don't understand why num_pages, current_page, and start were all net additions.  Didn't you just extend DefaultPagination?  Not a blocker.

Reply:
The default pagination was probably the vanilla DRF page-number paginator set via settings file.  Adam's new paginator extends the edX-DRF default paginator: https://github.com/openedx/edx-drf-extensions/blob/master/edx_rest_framework_extensions/paginators.py